docs/navigation-parameters-reference.md

```diff
@@ -270,6 +270,14 @@ settings: {
     }
     ```
 
+### pageErrorHandler
+- **type**: object
+- **description**: gives you the possibility to handle a situation if Luigi client doesn't respond. By default, it will redirect to the home page if nothing else is specified.
```

Comment:
```suggestion
- **description**: gives you the possibility to handle a situation in which Luigi Client doesn't respond. By default, it will redirect to the home page if nothing else is specified.
```